src/index.html

```diff
@@ -6,9 +6,65 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#page_top">
+        <img src="images/logo.png" alt="moyo">
+      </a>
+      <nav>
+
+        <ul class="nav__items">
+          <li class="nav__item">
+            <a
+              class="is-active a__item"
+              href="#apple"
+            >
+              APPLE
+            </a>
+          </li>
```

Comment:
Fix naming classes with BEM everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              class="is-active nav__link"
              href="#apple"
            >
              APPLE
            </a>
          </li>
```